src/app/code/community/Omise/Gateway/etc/config.xml

```diff
@@ -135,6 +143,14 @@
                 <model>omise_gateway/payment_offsitealipay</model>
                 <title>Alipay</title>
             </omise_offsite_alipay>
+
+            <omise_offsite_instalment>
+                <active>0</active>
+                <payment_action>authorize_capture</payment_action>
+                <model>omise_gateway/payment_offsiteinstalment</model>
+                <title>Instalment payments</title>
```

Comment:
@jonrandy I vote to have only `Installment` just  to make it consistent with the others.

Reply:
Will let you know when change is done

Reply:
Done - this is going to be a nightmare for my British brain going forward

Reply:
:))))))))